### DISCUSSION TITLE
Disable Timewarp and photo?

### DISCUSSION THREAD

**Discussion by RasmusTQ** [gopro/labs#516] [General]
I remember I've seen a feature where I can disable switching mode on short-pressing the power button. I'm setting up a bunch of gopros for non camera people to use and would love to have the camera just turn on and capture video, without the risk of them accidentally making time lapses of anything. I've searched the manual and forums but can't come up with the right terminology. Super thankful for any pointers in the right direction!

**Comment by dnewman-gpsw** [comment]
This the feature, weirdly named as it was designed for slightly different application. https://gopro.github.io/labs/control/archive/

**Comment by RasmusTQ** [comment]
Thanks a lot David!
